[REWORK] SetEntityAlpha (#285)
citizenfx/natives: ENTITY/SetEntityAlpha.md

```c
// 0x44A0870B7E92D7C0 0xAE667CB0
void SET_ENTITY_ALPHA(Entity entity, int alphaLevel, bool skin);
```

> ```
> skin - everything alpha except skin

Reviewer: ```suggestion
```

> Set entity alpha level. Ranging from 0 to 255 but changes occur after every 20 percent (after every 51).

Reviewer: ```suggestion
This native sets the entity's alpha level.
```

> ```

Reviewer: ```suggestion
```

## Parameters

- **entity**: The entity to change the alpha for.

> - **alphaLevel**: The alpha level between 0-255.

Reviewer: ```suggestion
- **alphaLevel**: The alpha level ranges from 0 to 255, but changes occur every 20% (every 51).
```

> - **skin**: Whether to not change alpha level of the entity skin.

Reviewer: ```suggestion
- **skin**: Whether or not to change the alpha of the entity's skin.
```
